repostary_card/models/card.py

```diff
@@ -0,0 +1,35 @@
+from odoo import models, fields, api
+from datetime import datetime, date
+
+class Card(models.Model):
+    _name = 'card'
+    _rec_name = 'beneficiary'
+    
+    card_number = fields.Char(string='Card Number' , readonly=True)
+    beneficiary =  fields.Many2one('res.partner', string='Beneficiary',)
+    status =  fields.Selection( string='Status',selection=[('value1', 'active'),('value2', 'expired'), ('value3', 'not_active')], default='value1')
```

Comment:
@mohammed-almaqdy you should use small case as the value and capital case as showing string rather than value1...